app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
   package="org.oppia.app">
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
We shouldn't be shipping with this permission. It also won't work on newer devices since you need to explicitly request the permission.

With Bazel, we can stage the test-only builds to programmatically add this permission to avoid adding it to the top-level manifest. In Gradle I don't think we can do that. Given this, I suggest that you:
1) File an issue for conditionally setting up external storage permission via Bazel (for post-M3)
2) Remove this from this PR
3) Write up a document explaining how to run & setup the screenshot tests. One step in there will need to be adding this permission back.

(1) will block being able to run screenshot tests in CI.

Reply:
Done, updated the description.

Reply:
@MohamedMedhat1998 Please mention the issue number in this comment.

Reply:
@rt4914 Ok.  
Issue #1746